src/components/RegionList/RegionListComponent.tsx

```diff
@@ -152,6 +152,15 @@ export class RegionListComponent extends React.Component<WidgetProps> {
         DialogStore.Instance.showDialog(DialogId.Region);
     };
 
+    private handleRegionDeleteAllClicked = () => {
```

Comment:
Would be good to move the function to `AppStore` so that we can use it with scripting.